samples/snippets/csharp/concepts/linq/query-expression-basics_7.cs

```diff
@@ -0,0 +1,27 @@
+            int highestScore =
+                (from score in scores
+                 select score)
+                .Max();
+
+            // or split the expression
+            IEnumerable<int> scoreQuery =
+                from score in scores
+                select score;
+
+            int highScore = scoreQuery.Max();
```

Comment:
The answer is the same, but this query expression:

```csharp
int highestScore = 
    (from score in scores
     select score).Max;
```

Compile to:

```csharp
scores.Select(s => s).Max();
```
instead of:
```csharp
scores.Max()
```